### DISCUSSION TITLE
2022/12/18/increasing-mastodon-post-length/

### DISCUSSION THREAD

**Discussion by giscus** [jdeluyck/kcoreorg-website#146] [Blog Comments]
# 2022/12/18/increasing-mastodon-post-length/

Mastodon comes with a default post size of 500 characters. This is not set in stone, and can be increased fairly easily (last tested on Mastodon 4.0.2):

https://kcore.org/2022/12/18/increasing-mastodon-post-length/

**Comment by DalekDave** [comment]
I followed the instructions carefully with Mastodon v4.2.0-beta1 but this hack no longer works, unfortunately. The file live/app/javascript/mastodon/features/compose/components/compose_form.js is now a .jsx file.
I'd really like to increase the max post length, could you figure out an update for this article?
Thanks anyway!

**Comment by DalekDave** [comment]
My previous post was incorrect. The above hack does still work. I just have Node.js version v18.17.1 with Mastodon v4.2.0.

The Mastodon team gave me the solution:

"Node 18 is not supported by our frontend toolchain out of the box."

You need to either to use Node 16, or to use the command: 

NODE_OPTIONS=--openssl-legacy-provider RAILS_ENV=production bundle exec rails assets:precompile.

To force your assets to be compiled again, you can execute the following command before the above precompilation command: 

rm tmp/cache/webpacker/last-compilation-digest-production
